PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/core/FunctionUtils.java

```diff
@@ -164,10 +164,13 @@ public static void copyLocalSettingsToStagingFolder(Path stagingFolder, Path loc
 
     public static void prepareStagingFolder(Path stagingFolder, Path hostJson, Module module, PsiMethod[] methods) throws AzureExecutionException, IOException {
         final Map<String, FunctionConfiguration> configMap = generateConfigurations(methods);
+        if (stagingFolder.toFile().isDirectory()) {
```

Comment:
Do we need add validation that throw exception when stagingFolder is null or not a file, then clean the folder directly?

Reply:
if stagingFolder is null, the NPE will be thrown, it is expeced.
if it doesn't exist, buildJarFileToStagingPath will create it